style.css

```diff
@@ -240,3 +249,233 @@ ul .languages {
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.box_Style {
+  margin: auto 24px 24px 24px;
+  padding: 15px 16px;
+  width: 327px;
+  height: 48px;
+}
+
+.form_items textarea {
+  width: 327px;
+  height: 114px;
+  padding: 12px;
+  background: #fff;
+  border: 1px solid #60c095;
+  border-radius: 4px;
+}
+
+.form_items #get_in_touch {
+  margin: 24px auto auto auto;
+  padding: 12px;
+  font-size: 17px;
+  color: #fff;
+  font-weight: 500;
+  border: none;
+}
+
+.foot_con p {
+  color: #42526e;
+  font-size: 20px;
+  text-align: center;
+  line-height: 32px;
+  margin: 0 auto 147px auto;
+}
+
+footer .social_media_icons {
+  margin-bottom: 19px;
+}
+
+.break_line {
+  display: none;
+}
+
+/* Media queries */
+
+@media screen and (min-width: 769px) {
+  /*  Header Section */
+
+  .header {
+    display: none;
+  }
+
+  .navbar_destop_ver {
+    background-color: transparent;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: absolute;
+    top: 50px;
+  }
+
+  .navbar_destop_ver a {
+    text-decoration: none;
+    color: #172b4d;
+    font-weight: 600;
+    margin: auto 40px;
+  }
+
+  /* Headline Section */
+
+  .headline {
+    background-image: url("./images/Header-llustration-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 1300px;
+    margin-top: 0;
+  }
+
+  .whoiam {
+    line-height: 60px;
+    font-size: 48px;
+    margin-top: -350px;
+  }
+
+  .job {
+    margin-top: 25px;
+  }
+
+  .abt_me {
+    margin-top: -10px;
+    margin-bottom: 250px;
+    padding-left: 355px;
+    padding-right: 355px;
+  }
+
+  section .social_media_icons {
+    margin-top: -200px;
+  }
+
+  /* Works Section */
+
+  hr {
+    display: none;
+  }
+
+  .cards {
+    grid-template-rows: repeat(2, 600px);
+    grid-template-columns: repeat(3, 369px);
+    justify-content: center;
+    align-content: center;
+  }
+
+  .card {
+    width: 369px;
+  }
+
+  /* About Me Section */
+
+  .card2 {
+    grid-template-columns: repeat(3, auto);
+    column-gap: 24px;
+  }
+
+  .detailed_info {
+    width: 100%;
+    height: 331px;
+    background-image: url("./images/Frame.svg"), url("./images/illustration-about-me-2-desktop.svg");
+    background-position: right top, left bottom;
+    background-repeat: no-repeat, no-repeat;
+    margin: 40px auto;
+  }
+
+  .detailed_info p {
+    margin: auto 405px;
+  }
+
+  .detailed_info h2 {
+    margin-top: 70px;
+  }
+
+  /* Contact Form */
+
+  .contact_form {
+    width: 1650px;
+    height: 900px;
+    position: relative;
+    overflow: visible;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    margin: -170px 290px;
+    background: url("./images/Frame2.svg") no-repeat, url("./images/illustration-2-contact-form-desktop.svg") no-repeat, url("./images/Frame3.svg") no-repeat;
+    background-position: 0 580px, 170px 650px, 1100px 400px;
+    background-size: 9%, 8%, 12%;
```

Comment:
- [ ] Kindly fix the background image in the contact form it should look like the one in the template.
### Your design
![Screenshot from 2021-12-03 14-23-36](https://user-images.githubusercontent.com/65068771/144602072-c73c8201-0d18-4713-ac68-5a478abceb29.png)

### The template
![Screenshot from 2021-12-03 14-22-50](https://user-images.githubusercontent.com/65068771/144602107-3b6ad194-8d15-42c8-8816-4b0b2b832f61.png)

Reply:
Please can clarify the issue here.  I believe the background looks exactly the same as the design.

Reply:
Hello @TimmyChan99

Reply:
Hey @andersonlebon